src/main/java/cn/nukkit/level/Level.java

```diff
@@ -2080,33 +2080,34 @@ private void addBlockChange(long index, int x, int y, int z) {
         }
     }
 
-    public void dropItem(Vector3 source, Item item) {
-        this.dropItem(source, item, null);
+    public EntityItem dropItem(Vector3 source, Item item) {
+        return this.dropItem(source, item, null);
     }
 
-    public void dropItem(Vector3 source, Item item, Vector3 motion) {
-        this.dropItem(source, item, motion, 10);
+    public EntityItem dropItem(Vector3 source, Item item, Vector3 motion) {
+        return this.dropItem(source, item, motion, 10);
     }
 
-    public void dropItem(Vector3 source, Item item, Vector3 motion, int delay) {
-        this.dropItem(source, item, motion, false, delay);
+    public EntityItem dropItem(Vector3 source, Item item, Vector3 motion, int delay) {
+        return this.dropItem(source, item, motion, false, delay);
```

Comment:
The infringing changes.

Reply:
So shall I create a new method to it and revert changes on Level#dropItem return values?

Reply:
Yes. Check if you need to reference the `EntityItem` later and if those new methods are needed.